### PR TITLE
Enable Travis CI

### PR DESCRIPTION
**Pull request by hroncok**

### CODE REVIEW

**Comment by hroncok** [comment]
I've also included a fix of #6 in this PR to make the tests pass.

**Comment by junaruga** [comment]
@hroncok thanks! let me see the code to approve the review.

**Comment by hroncok** [comment]
I got an idea for one slight change.

**Comment by hroncok** [comment]
It worked. Amended. Ready for review.

**Comment by junaruga** [comment]
@hroncok I checked your code. I commented on the code to ask you. Could you check those? Thanks.

**Comment by hroncok** [comment]
I cannot see any comments. Aren't those comments *Pending*?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+python:
+- '3.6'
+- '3.5'
+- '3.4'
+
+matrix:
+  include:
+    - python: 3.6
+      sudo: required
+      dist: trusty
```
Comment (hroncok) [inline .travis.yml]: Trusty is a version of Ubuntu. Currently Travis offers "very old Ubuntu" by default and "old Ubuntu" by choosing trusty here. I wen't with trusty as I'm going to do further hacks and was as new ubuntu as possible.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      env:
+      - TOXENV=style
+      before_install:
+      - sudo sed -i 's/trusty/yakkety/' /etc/apt/sources.list
+      - sudo rm -rf /etc/apt/sources.list.d/*
```
Comment (hroncok) [inline .travis.yml]: Line 1: It's changing the Ubuntu release code name in the APT sources list, so when I install `spellcheck`, the version from Yakkety is installed. (Trusty have one version in backports, but that one is old and reports some false errors.)

Line 2: Removing any 3rd party repositories available on Travis CI (not needed, proven to make trouble when doing Line 1).

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      - sudo apt-get -qq update && sudo apt-get -qq install shellcheck
+
+cache:
+  directories:
+  - .cache/pip
```
Comment (hroncok) [inline .travis.yml]: Should have been `$HOME/.cache/pip`, sorry about that. Good catch.

Comment (hroncok) [inline .travis.yml]: Amended.

**Comment by hroncok** [comment]
Thanks. I've made it so that Travis CI now has to success before merging.

**Comment by junaruga** [comment]
@hroncok Yes I will follow the way for other PRs. We were able to be ready to sent PRs.

**Pull request merged**